juniper_official/Kernel/check-dead-ifstate-client.rule

```diff
@@ -0,0 +1,50 @@
+/*
+ * This rule checks if there are any dead ifstate clients not cleaned up in the
+ * system for the last 20 minutes.
+ *
+ * By default, it checks if all the samples collected every 1 minute are
+ * non-zero for contiguous 20 minutes.
+ */
+iceberg {
+    topic kernel.ifstate {
+        rule check-dead-ifstate-client {
+            sensor kernel-ifstate-sensor {
+                open-config {
+                    sensor-name /junos/kernel-ifstate;
+                    frequency 1m;
+                }
+            }
+            field dead-clients-cnt {
+                sensor kernel-ifstate-sensor {
+                    path /junos/kernel-ifstate/dead-clients-cnt;
+                }
+                type integer;
+                description "Dead ifstate clients count in the system";
+            }
+            trigger report-dead-ifstate-clients {
```

Comment:
1) Provide trigger description and synopsis detailing sensor info.
2) Trigger frequency is must as default frequency is 10s and trigger gets evaluated once in 10s. but here sensor frequecy is 10 mins hence trigger frequency can be greaterthan equal to 10 mins.
e.g. synopsis "Routing engine GRES kpi";
       description "sets health based on Routing engine CPU utilization for 15 minutes";

Reply:
Addressed